File: task-03/HelloWorld.java
import java.util.ArrayList;
import java.util.List;
import java.util.Scanner;

public class PrimeNumbers {
    public static boolean isPrime(int num) {
        if (num < 2) return false;
        for (int i = 2; i <= Math.sqrt(num); i++) {
            if (num % i == 0) return false;
        }
        return true;
    }

    public static List<Integer> findPrimes(int n) {
        List<Integer> primes = new ArrayList<>();
        for (int num = 2; num <= n; num++) {
            if (isPrime(num)) primes.add(num);
        }
        return primes;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int n = scanner.nextInt();
        List<Integer> primeNumbers = findPrimes(n);
        System.out.println("Prime numbers up to " + n + " are: " + primeNumbers);
        scanner.close();
    }
}

File: task-03/HelloWorld.rb
def is_prime(num)
    return false if num < 2
    (2..Math.sqrt(num)).each do |i|
        return false if num % i == 0
    end
    true
end

def find_primes(n)
    primes = []
    (2..n).each do |num|
        primes << num if is_prime(num)
    end
    primes
end

print "Enter a number: "
n = gets.chomp.to_i
puts "Prime numbers up to #{n} are: #{find_primes(n)}"
